### DISCUSSION TITLE
Discord.js embedded message is returning empty

### DISCUSSION THREAD

**Discussion by Piesrgr8** [discordjs/discord.js#7417] [Q&A] [answered]
I am on the latest version as of now and I am very burnt out. Right now, it is returning an empty message.

I have it set up to do this:

`async function getStream() { `
`const query = await twitch.getStreams({channel: "Twitch_Channel"});`
`if (typeof query !== 'undefined') {`
`    const stream = query.data[0];`
`    const embed = new Discord.MessageEmbed()`
`        .setColor('PURPLE')`
`        .setTitle("I am now LIVE!")`
`        .setDescription("We trying here")`
`        .setURL("https://www.twitch.tv/Twitch_Channel");`
`    client.channels.cache.get('CHANNEL_ID').send({embed: embed});`
`    } else {`
`        console.log(query + " Twitch stream offline. Searching again in 3...");`
`    }`
`} `

This is sent to a client.on('messageCreate') for now, because tutorial after tutorial, I came to the conclusion that it needs a message parameter. The goal is to NOT have it trigger on message creation, but on a task timer.

I understand it'll need logic to determine if a message has been sent or not, but I will need to see if the message works first.

**Comment by DaStormer** [comment, marked as answer]
There's no `embed` parameter in the message options anymore. You would need to use `embeds` and it'll only accept arrays.

so in your code, it should look like this: `.send({ embeds: [embed] })`

if you recently updated to v13 from v12 I'd recommend reading this: https://discordjs.guide/additional-info/changes-in-v13

**Reply by Piesrgr8** [reply]
I had no clue that making embed plural and making it an array would solve the issue. For that, I thank you lol
